spec/inject_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'inject'
+
+describe Array do
+  it "can add" do
+    expect([1,2,3].inject {|m,e| m + e} ).to eq 6
+    expect([1,2,3].my_inject {|m,e| m + e} ).to eq 6
+  end
+
+  it "can times" do
+     expect([1,2,3].inject {|m,e| m * e} ).to eq 6
+     expect([1,2,3].my_inject {|m,e| m * e} ).to eq 6
+  end
+
+  it "can add with a stated starting point" do
+     expect([1,2,3,4,5].inject(2) {|m,e| m + e} ).to eq 17
+     expect([1,2,3,4,5].my_inject(2) {|m,e| m + e} ).to eq 17
```

Comment:
Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.<br>Space inside parentheses detected.